### PR TITLE
Text describing the parallelism allowed in a P4Runtime server

### PR DESCRIPTION
**Pull request by jafingerhut**
... when serving ReadRequest and WriteRequest messages.

### CODE REVIEW

**Comment by jafingerhut** [comment]
This new proposed text is intended to address this issue: https://github.com/p4lang/p4runtime/issues/199

I was not sure of the best place to add such text.  Right now this PR has it added in a new subsection at the end of the "Read RPC" section.

File: docs/v1/P4Runtime-Spec.mdk [hunk 1]
```diff
@@ -4496,6 +4496,51 @@ multi-reader lock (also known as multiple-readers/single-writer lock).
...
+  processing, or `W` has not yet begun processing.
+
+For example, if a P4Runtime server maintained, independently for each
+device it managed, a separate multi-reader single-writer lock for each
+stateful object in the P4 program, and before starting the processing
+of a `WriteRequest` message it acquired a write lock for each stateful
+object affected by the `WriteRequest`, and before starting the
+processing of a `ReadRequest` message it acquired a read lock for each
+stateful object accessed by the `ReadRequest`, such an implementation
+should meet all of the requirements above.
```
Comment (stefanheule) [inline docs/v1/P4Runtime-Spec.mdk]: Why does it say "should meet" instead of "meets"?  Is there anything else to be considered?

Comment (jafingerhut) [reply docs/v1/P4Runtime-Spec.mdk]: The only reason it says that is my often-cautious way of writing when I'm not yet sure of something.  Changing it to "meets" sounds like a good idea.  We can correct it later if I messed it up :)

Comment (stefanheule) [reply docs/v1/P4Runtime-Spec.mdk]: No worries, thank for updating and writing all of this in the first place.

File: docs/v1/P4Runtime-Spec.mdk [hunk 2]
```diff
@@ -4496,7 +4496,7 @@ multi-reader lock (also known as multiple-readers/single-writer lock).
 Conversely (&eg; in a single-threaded architecture), it may choose to serialize
 `Read` RPC processing.
 
-## Parallelism of read and write requests
+## Parallelism of Read and Write requests
```
Comment (antoninbas) [inline docs/v1/P4Runtime-Spec.mdk]: Actually that's not specific to "read" and "write", we capitalize every significant word in titles: https://github.com/p4lang/p4runtime/tree/master/docs#what-to-capitalize

Comment (jafingerhut) [reply docs/v1/P4Runtime-Spec.mdk]: Doh!  If I had read your comment more carefully, I would have noticed that.  Fixed now.

**Pull request merged**